src/filters/alpha/AlphaFilter.js

```diff
@@ -0,0 +1,45 @@
+import * as core from '../../core';
+import { readFileSync } from 'fs';
+import { join } from 'path';
+
+/**
+ * Simplest filter - applies alpha. Very handy.
+ *
+ * @class
+ * @extends PIXI.Filter
+ * @memberof PIXI.filters
+ */
+export default class AlphaFilter extends core.Filter
+{
+    /**
+     *
+     */
+    constructor()
+    {
+        super(
+            // vertex shader
+            readFileSync(join(__dirname, '../fragments/default.vert'), 'utf8'),
+            // fragment shader
+            readFileSync(join(__dirname, './alpha.frag'), 'utf8')
+        );
+
+        this.uniforms.uAlpha = 1.0;
```

Comment:
why not do `this.alpha = 1.0` here instead? It's shorter and doesn't duplicate `uAlpha`